fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/RecordQueryPlannerConfiguration.java

```diff
@@ -39,6 +39,11 @@
  */
 @API(API.Status.MAINTAINED)
 public class RecordQueryPlannerConfiguration {
+    /**
+     * An indicator as to whether to use index prefetch for a given query.
+     */
+    public enum IndexPrefetchUse { NONE, USE_INDEX_PREFETCH, USE_INDEX_PREFETCH_WITH_FALLBACK }
```

Comment:
Done. Renamed to "Remote Fetch" across the board (and the enum values). It felt more correct to eliminate the "prefetch" consistently across the method names and comments rather than just the enum values. If that was not your intent, please let me know.

Reply:
Yes, great! That's what I was thinking (that is, of a general feature rename). Thanks!